af/.gitignore

```diff
@@ -0,0 +1,6 @@
+# TMP
+airflow/dumb-init_1.2.0_amd64.deb
+shovel/*.flame
+shovel/*.svg
+shovel/bridge_db.json
+useless-test-dags/
```

Comment:
Were these files ever commited?...

Reply:
Don't think so, but I had them lying around on my filesystem so I populated the gitignore with them